keras/engine/training.py

```diff
@@ -1399,9 +1399,9 @@ def _test_loop(self, f, ins, batch_size=None, verbose=0, steps=None):
             return outs[0]
         return outs
 
-    def _standardize_user_data(self, x, y,
-                               sample_weight=None, class_weight=None,
-                               check_batch_axis=True, batch_size=None):
+    def _standardize_training_data(self, x, y,
```

Comment:
This is not training data, since it is used in `evaluate`/etc. Proper name is `_standardize_user_data`